packages/entities/entities-plugins/docs/plugin-select.md

```diff
@@ -92,21 +92,19 @@ A grid component for selecting Plugins.
 
 The base konnect or kongManger config.
 
-#### `disableCustomPlugins`
+#### `customPluginSupport`
 
-- type: `boolean`
+- type: `false | 'disabled' | 'schema' | 'streaming'`
```

Comment:
Do you think it will be better to use `'hidden'` or `'none'` instead of keeping the boolean type here?